travis/install-common-deps.sh

```diff
@@ -28,21 +26,23 @@ python --version
 pip install --upgrade pip==19.3.1
 # Install Python test dependencies only if we're running Python tests
 if [[ "$INSTALL_SMALL_PYTHON_DEPS" == "true" ]]; then
-  pip install -r ./travis/small-requirements.txt
+  pip install --quiet -r ./travis/small-requirements.txt
```

Comment:
Github Actions isn't as good about large output, so made some of these things a bit quieter.